src/textual/cli/cli.py

```diff
@@ -172,6 +172,14 @@ def run_app(import_name: str, dev: bool, press: str) -> None:
 @run.command("borders")
 def borders():
     """Explore the border styles available in Textual."""
-    from ..devtools import borders
+    from textual.cli.previews import borders
```

Comment:
Thought it made sense to keep these modules together like this, rather than the previews/demo apps being jumbled in with devtools code.